app/assets/styles/admin.scss

```diff
@@ -7,7 +7,7 @@
   padding: .5em;
 }
 
-#editor-about-admin, #editor-contact-admin {

```

Comment:
pull and merge in master - I changed this in my latest fix for the editing bug
Basically we no longer use `#editor-inserttype-admin` format but instead all the admin side editors will just have a `class="editor-admin"` and all the user side editors will have `class="editor-user"` for styling